src/common/libschedutil/hello.c

```diff
@@ -14,24 +14,30 @@
 #include <flux/core.h>
 #include <jansson.h>
 
+#include "init.h"
 #include "hello.h"
 
 static int schedutil_hello_job (flux_t *h, flux_jobid_t id,
                                 hello_f *cb, void *arg)
 {
     char key[64];
     flux_future_t *f;
-    const char *s;
```

Comment:
suggestion: go with something other than `s` as the context variable.  `s` is often a temporary string in our code base (for better or worse) as you encountered here.  Maybe `util`, `su`, or `ctx`?  Then use that consistently.

Reply:
Changed in https://github.com/flux-framework/flux-core/pull/2226/commits/b412ff1c309cf73a70c9f1e677f18446045ffddb